corehq/apps/reports/standard/deployments.py

```diff
@@ -165,13 +168,15 @@ def selected_app_id(self):
         return self.request_params.get(SelectApplicationFilter.slug, None)
 
     @quickcache(['app_id'], timeout=60 * 60)
-    def get_app_name(self, app_id):
+    def _get_app(self, app_id):
```

Comment:
caching the whole app just for the profile name seems excessive. Can we rather switch this to just caching the name.

Reply:
@snopoke updated this caching so that we have the app available when fetching build profile name later.

Reply:
Yes I saw that but what I'm saying is that you're putting MB of data in the cache just for the name. Rather just cache the name.

Reply:
okay, i will revisit that
1. i will memoize just the name, and then just may be keep a dict mapping of app_id to build_profiles

Reply:
Sorry for the confusion here @snopoke 
i read `profile name` as `app name` and misunderstood.